src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -204,13 +208,20 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
     }
     const startDate = this.entryForm.value.start_date;
     const endDate = this.entryForm.value.end_date;
+    this.starDateValue = this.entryForm.value.start_hour === this.starDateValue ?
+      this.entryToEdit.start_date :
+      new Date(`${startDate}T${this.entryForm.value.start_hour.trim()}`).toISOString();
+    this.endDateValue = this.entryForm.value.end_hour === this.endDateValue ?
+      this.entryToEdit.end_date :
+      new Date(`${endDate}T${this.entryForm.value.end_hour.trim()}`).toISOString();
```

Comment:
I see an opportunity to extract lines 213 and 216 into a method that receives the `entryForm.value.X_VALUE` and returns everything that is being done in that large sentence.

If you find that in other files, could be an utilitary. :)

Reply:
It's Done thanks.